pkg/processresetqueue/reset.go

```diff
@@ -19,7 +19,6 @@ type ResetInput struct {
 	ResetQueueURL *string
 	ResetBuild    common.Builder
 	BuildName     *string
-	DbSvc         db.DBer
```

Comment:
We no longer need to read from the DB as the entire record is in the queue.

Reply:
👍 Not only do we read the entire record, but it's a rather important point that _we're reading the record as a snapshot at the time the record was written_. This is rather a critical point if we're writing events that can be played back or are idempotent.